### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by TheEmperor342**

### CODE REVIEW

**Comment by itsmebirdie** [comment]
PR acknowledged, review in progress

File: Cogs/Welcomer.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            with open("WelcomerConfig.json",'w') as f: f.write(json.dumps(self.CONFIG))
+            
+            await ctx.send(embed=discord.Embed(color=discord.Color.green(),description=f":white_check_mark: Welcome Message Updated!\nSet to: {msg.replace('|user|',ctx.author.mention).replace('|guild|', str(ctx.guild))}",title="WELCOMER"))
+            
+    #############################################################################################
+    
+    ## ==> TO SET LEAVE MESSAGE
+    #############################################################################################
+    
+    ## not done yet lel
```
Comment (itsmebirdie) [inline Cogs/Welcomer.py:82]: be professional with comments even if it is under development

File: config.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{ "token": "no peeking" }
```
Comment (itsmebirdie) [inline config.json:1]: in the next PR, make the token be in a system environment file i.e, `.env` file.

File: Cogs/Welcomer.py (hunk 1 above)
Comment (TheEmperor342) [reply Cogs/Welcomer.py:82]: I am a bit informal with this kinda stuff 😂

**Review by itsmebirdie — APPROVED** [review]
### Code Quality
The code quality is very good except at parts where things could go messy, please try to add comments where a decent explanation is needed. Overall code quality was good and approved
### Code Comments
Just for records, be professional while writing comments especially at those parts where development is still going on, also please check the in-code comments I have written so to be acknowledged by you.
### Side Notes
Overall, the commits are reviewed and checked and are ready for merge, will still request to improve the comments and as a main sidenote, try to keep the tokens, keys and environment variables in a separate system environment i.e, `.env` file, check the in-code comments for better visualization on the issue

File: Cogs/Welcomer.py (hunk 1 above)
Comment (itsmebirdie) [reply Cogs/Welcomer.py:82]: You have to be professional, these are the rules

Comment (TheEmperor342) [reply Cogs/Welcomer.py:82]: alright

**Pull request merged**